### PR TITLE
[stable27] 40289 fix appearance

### PR DESCRIPTION
**Pull request by nfebe**
* Resolves: https://github.com/nextcloud/server/issues/40289

| Before | Currently |  Without Icons |
| --------- | ------------ | -------------- |
|2023-09-06 12_10_41-Shared with others - Nextcloud – Mozilla Firefox |Screenshot from 2023-09-08 00-23-51 | Screenshot from 2023-09-08 00-18-11 |

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
@juliushaertl @Fenn-CS nice 👍 Can we left-align the content so we are closer to the mockups and v28 as described in https://github.com/nextcloud/server/issues/26691 ?

**Comment by Fenn-CS** [comment]
> @juliushaertl @Fenn-CS nice 👍 Can we left-align the content so we are closer to the mockups and v28 as described in https://github.com/nextcloud/server/issues/26691 ?

That was the default, option and I even got a screenshot for it earlier, I just forgot to add it here. The reason, I centered it with the icons, is that, I realized it was centered without them on 27... But yeah done.

Screenshot from 2023-09-08 00-12-15

cc: @AndyScherzinger

**Review by AndyScherzinger — APPROVED** [review]

File: apps/files_sharing/src/views/SharingDetailsTab.vue [hunk 1]
```diff
@@ -951,11 +946,25 @@ export default {
...
+							flex: 10%;
+						}
+
+						&:last-of-type {
+							flex: 90%
+						}
 					}
+
+					small {
+                       flex-basis: 100% !important;
```
Comment (skjnldsv) [inline apps/files_sharing/src/views/SharingDetailsTab.vue:965]:
```suggestion
						flex-basis: 100% !important;
```

File: apps/files_sharing/src/views/SharingDetailsTab.vue [hunk 2]
```diff
@@ -951,11 +946,25 @@ export default {
...
 						display: flex;
-						flex-direction: column;
+
+						&:first-of-type {
+							flex: 10%;
+						}
+
+						&:last-of-type {
+							flex: 90%
+						}
```
Comment (skjnldsv) [inline apps/files_sharing/src/views/SharingDetailsTab.vue:961]: Ideally this should be done in the vue components with a proper slot like you did initially :)

**Review by artonge — APPROVED** [review]

**Review by jancborchardt — COMMENTED** [review]
Just design feedback, not blocking:
- [x] The subline alignment of "Create, update, delete, etc." is too far to the left, it should ideally be left-aligned with the "Custom permissions" text as per the mockup. If the subline is too long it should ellipsize after 1 line

File: apps/files_sharing/src/views/SharingDetailsTab.vue (hunk 2 above)
Comment (Fenn-CS) [reply apps/files_sharing/src/views/SharingDetailsTab.vue:961]: Yeah @skjnldsv, this changes where already made in the component, but where not back-ported (not sure why), so went this route.

**Comment by Fenn-CS** [comment]
> Just design feedback, not blocking:
> 
>     The subline alignment of "Create, update, delete, etc." is too far to the left, it should ideally be left-aligned with the "Custom >permissions" text as per the mockup. If the subline is too long it should ellipsize after 1 line

Screenshot from 2023-09-11 13-25-59



cc: @jancborchardt

**Review by jancborchardt — APPROVED** [review]
Nice! :)

**Review by sorbaugh — APPROVED** [review]

**Pull request merged**